src/Symfony/Component/Form/Extension/Core/Type/TimezoneType.php

```diff
@@ -72,7 +116,7 @@ public function loadChoiceList($value = null)
             return $this->choiceList;
         }
 
-        return $this->choiceList = new ArrayChoiceList($this->getTimezones(), $value);
+        return $this->choiceList = new ArrayChoiceList($this->getTimezones($this->regions), $value);
```

Comment:
It basically has not purpose anymore.. not sure we should preserve its api in any way (i.e. forward to the callback loader or so).